docs/python_support/quickstart.md

```diff
@@ -0,0 +1,233 @@
+
+```bash
+echo '
+input_stream: "OVMS_PY_TENSOR:text"
+output_stream: "OVMS_PY_TENSOR:uppercase"
+
+node {
+  name: "pythonNode"
+  calculator: "PythonExecutorCalculator"
+  input_side_packet: "PYTHON_NODE_RESOURCES:py"
+  input_stream: "INPUT:text"
+  output_stream: "OUTPUT:uppercase"
+  node_options: {
+    [type.googleapis.com / mediapipe.PythonExecutorCalculatorOptions]: {
+      handler_path: "/models/python_model/model.py"
+    }
+  }
+}' >> models/python_model/graph.pbtxt
+```
+
+Above configuration file creates a graph with a single Python node that uses `PythonExecutorCalculator`, set's inputs and outputs and provides your Python code location in `handler_path`. 
+The `input_side_packet` value is an internal field used by the model server to share resources between graph instances - do not change it. 
+
+### Step 4: Prepare Server Configuration File
+
+Last piece of configuration would be the model server configuration file. 
+Create `config.json` with the following content in `workspace/models`:
+
+```bash
+echo '
+{
+    "model_config_list": [],
+    "mediapipe_config_list": [
+    {
+        "name":"python_model",
+        "graph_path":"/models/python_model/graph.pbtxt"
+    }
+    ]
+}
+' >> models/config.json
+```
+
+This tells OVMS to to serve the graph under given name `python_model`.
+
+### Step 5: Deploy OpenVINO Model Server
+
+Before running the server let's check if all files required for deployment are in place. Check the contents of `workspace/models` catalog as it will be mounted to the container:
+```
+tree models
+models
+├── config.json
+└── python_model
+    ├── graph.pbtxt
+    └── model.py
+```
+
+Now let's run the server:
+```bash
+docker run -it --rm -p 9000:9000 -v $PWD/models:/models openvino/model_server:latest --config_path /models/config.json --port 9000
+```
+
+### Step 6: Create Client Application
+
+Now that the Python model is deployed, you can focus on the other end - the client application. When writing the client keep in mind how the server side code looks like as they must be complementary.
+
+First let's connect to the server hosted on `localhost` with gRPC interface available on port `9000`:
+
+```python
+import tritonclient.grpc as grpcclient
+client = grpcclient.InferenceServerClient("localhost:9000")
+```
+
+You will send a string, so let's create one and encode it to UTF-8, because that's what the server side code expects:
+
+```python
+data = "Make this text uppercase.".encode()
+```
+
+Now let's pack that data into a gRPC structure that will be sent to the server:
+
+```python
+infer_input = grpcclient.InferInput("text", [len(data)], "BYTES")
+infer_input._raw_content = data
+```
+You've created InferInput object that will correspond to the servable input with name "text" (more about it soon), shape of (number_of_encoded_bytes) and datatype "BYTES". The data itself has been written to raw_content field. All of these values can be accessed on the server side.
```

Comment:
"shape of (number_of_encoded_bytes)" - is it missing something? or "of" is not required